developer_manual/digging_deeper/groupware/calendar.rst

```diff
@@ -161,6 +161,29 @@ Calendars that only return `ICalendar` are implicitly read-only. If your app's c
 
     }
 
+Handling iMIP data 
+~~~~~~~~~~~~~~~~~~
+
+You may implement the ``IHandleIMipMessage`` interface to process iMIP data you receive in a client and want to pass on for processing to the backend. 
+
+Please be aware that there are some security considerations to take into account.
```

Comment:
Can we also link to some resources about the security considerations? Is this related to the next paragraph with the RFC link? Then remove the separating line breaks.

Reply:
Yes it is.